src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,72 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are three mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- Seconding
+  - A proposal is second'ed when a team member formally endorses the proposal. It is intended that
+    seconding only occur once discussion has concluded and team members have had time to raise
+    concerns. Seconding tentatively accepts a proposal subject to a ten-day waiting period for
+    other team members to raise any concerns.
+- FCP
+  - A final comment period will require sign-off from a majority of the rustdoc team to approve
+    a proposal and then a ten day waiting period.
+  - FCPs can be used to approve any form of proposal.
+
+## Proposals
+There are two ways to propose a change to the rustdoc team. The appropriate choice depends on
+the nature of the proposal, described below.
+
+- Request For Comments (RFC)
+  - RFCs are pull requests to the [`rust-lang/rfcs`][rfcs] repository and are a heavy-weight
+    proposal mechanism, reserved for significant changes.
+  - RFC proposals can only be approved by *FCPs*.
+- Pull Request (PR)
+  - PRs are pull requests to the [`rust-lang/rust`][rust] repository and are a light-weight
+    proposal mechanism, suitable for most proposals. PRs are preferred when the proposal is
+    accompanied by a small patchset (such as stabilization of a compiler flag or addition of
+    a new target).
+  - PR proposals can be approved by *FCPs* or *r+*.
+
+### When are FCPs/RFCs required?
+
+An FCP will be needed for any stabilization of small user-facing changes, like UI/UX changes, new
+command-line arguments, new attributes, etc. However, if the change is considered too big/important,
+an RFC will need to be written and approved before the change will be accepted.
```

Comment:
Ought to mention somewhere that:

"When opening an FCP, make sure only the relevant subteam is labeled on the issue, to avoid pinging people with changes they aren't interested in."